frontend/src/components/Graph.js

```diff
@@ -61,29 +71,44 @@ class Graph extends React.Component {
 
   render() {
     return (
-      <VictoryChart domainPadding={20} height={300}>
+      <VictoryChart
+        domainPadding={20}
+        height={300}
+        theme={VictoryTheme.material}
+      >
         <VictoryLabel
           text="Response Rates Data Over Collection Period"
-          x={225}
-          y={50}
+          x={170}
+          y={20}
           textAnchor="middle"
         />
         <VictoryAxis
           tickValues={this.state.xLabels}
           label="Days After Initial Census Mailing"
+          style={{ axisLabel: { padding: 37 } }}
         />
         <VictoryAxis
           dependentAxis
           tickValues={this.state.yLabels}
           label="Response Rate"
+          style={{ axisLabel: { padding: 35 } }}
         />
-
         <VictoryLine
           style={{
             data: { stroke: LINE_COLOR },
             parent: { border: BORDER }
           }}
           data={this.state.data}
+          animate={{
+            duration: 1000,
+            onLoad: { duration: 1000 }
+          }}
+          style={{
+            data: {
+              stroke: "gray",
+              strokeWidth: 2
+            }
```

Comment:
is it possible to move these styles to a .css file?

Reply:
it might be a little weird to move these out of the component since it's being passed in as props, so i think we should keep it in

Reply:
also same thing with the magic numbers